apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx

```diff
@@ -28,5 +35,27 @@ export const FirebaseFeatureFlagsProvider = ({ children }: Props): JSX.Element =
     }
   };
 
-  return <FeatureFlagsProvider getFeature={getFeature}>{children}</FeatureFlagsProvider>;
+  const overrideFeature = (key: FeatureId, value: Feature): void => {
+    const actualRemoteValue = getFeature(key, false);
+    if (!isEqual(actualRemoteValue, value)) {
+      const overridenValue = { ...value, overridesRemote: true };
+      setLocalOverrides(currentOverrides => ({ ...currentOverrides, [key]: overridenValue }));
+    } else {
+      console.error(`Not overriding `);
```

Comment:
I'm so sorry... but there's a trailing space here and it triggers me

Reply:
hahahaha, suggest an edit here, and we can commit it

Reply:
Oh didn't know we could, I did it